src/main/java/Event.java

```diff
@@ -0,0 +1,56 @@
+public class Event extends Task {
+    private final String start;
+    private final String end;
+
+    public Event(String taskName, String start, String end, boolean isCompleted) {
+        super(taskName, isCompleted);
+        this.start = start;
+        this.end = end;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from: " + this.start + " to: " + this.end + ")";
+    }
+
+    @Override
+    public String getStringRepresentation() {
+        return "event " + taskName + " /from " + this.start + " /to " + this.end + getIsCompletedString();
+    }
+
+    public static Event getTask(String currentInput)
+        throws EmptyTaskDescription, InvalidTaskArguments {
+        try {
+            boolean isCompleted = currentInput.contains(Task.IS_COMPLETED_STRING);
+            currentInput = currentInput.replaceAll(Task.IS_COMPLETED_STRING, "");
+
+            int idxOfStart = currentInput.indexOf(" /from ");
+            int idxOfEnd = currentInput.indexOf(" /to ");
```

Comment:
This magic string seems to be repeated in 18 as well, maybe consider declaring it as a const?